taskcluster/kinds/merge-corpus/kind.yml

```diff
@@ -13,18 +13,7 @@ transforms:
     - taskgraph.transforms.task:transforms
 
 kind-dependencies:
-    # This kinds handles two different parts of the graph. The first is `merge-corpus`, which
```

Comment:
those comments looked outdated since we now have only bicleaner ai and a separate kind for merge-devset